examples/heatTransfer/read/heatRead_adios2.cpp

```diff
@@ -109,7 +111,7 @@ int main(int argc, char *argv[])
     double *T = new double[vT->m_AvailableSteps * readsize[0] * readsize[1]];
```

Comment:
You can always make PrintData<T> a template function that takes const T*....PrintData should accommodate to the examples, not the other way around.